infra/core/ai/cognitiveservices.bicep

```diff
@@ -19,6 +30,8 @@
   properties: {
     customSubDomainName: customSubDomainName
     publicNetworkAccess: publicNetworkAccess
+    networkAcls: networkAcls
+    disableLocalAuth: disableLocalAuth
```

Comment:
That's what this PR does!

Reply:
fyi @tonybaloney Do you know why it's flagging? It should see the main.bicep passes in true, shouldn't it? I think this happened in other repo too, wonder if it's a bug in the tool.

Reply:
I think we need to configure the scan to only scan `main.bicep` and it'll automatically detect the parameters file. Otherwise it'll recursively scan the directory and then enumerate each combination of the parameters as a standalone file

Reply:
Hm, I think that's what I tried to do in the past and failed:
https://github.com/microsoft/security-devops-action/issues/47#issuecomment-1555360435
But maybe you can get it working.